bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -73,12 +83,15 @@ Please download a fresh copy of these files. If you have downloaded them for a p
 
 Download the [`learning_bash` directory](https://github.com/arcus/learning_bash) from GitHub. Once you go to the link:
 
-1. Click on the green **Code** button.
-2. Select **Download ZIP**
+1. Click on the green **Code** button. There are two "Code" buttons! The one directly below `arcus/learning_bash` functions more like a "home" button. One line below is a dropdown menu labeled `main` and to the right of `main` are three buttons: `Go to File`, `Add File`, and `Code`. This is the **Code** button you want.
```

Comment:
This might be sufficient information instead? Describing where it is on the screen gives more context to individuals who are colorblind, and the fact that the button is a drop-down is additional info that I think would also be useful to those using screenreaders. 

I think otherwise it gets a bit too text heavy and overwhelming, and also we'd need to add even more text to be accurate if we're trying to get exact location to screen readers (ie, they also have to go past "branch" and "tag" links on their way across the row). 

I'm only making this suggestion here, I figure we can workshop it and then once we are fully good with it, change it in all the other places in this PR as well. 

```suggestion
1. Click on the green **Code** drop-down button on the right-hand side of the screen. 
```

Reply:
I'm going to find a time for us to workshop this together because I think you are totally right that it is too text heavy, but I'm also worried because the **Code** button we want might be in the center of the screen, or even disappear entirely into a `...` menu if the browser window is narrow enough....

Reply:
change `on` to `towards`